app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/models/JsonFormatters.scala

```diff
@@ -95,10 +95,25 @@ trait ApiDefinitionJsonFormatters extends EndpointJsonFormatters with BasicApiDe
       case (version, status, Some(access), endpoints, endpointsEnabled) => ApiVersionDefinition(version, status, access, endpoints, endpointsEnabled)
     }
 
+  implicit val apiVersionSourceJF: Format[ApiVersionSource] = new Format[ApiVersionSource] {
+
+    def reads(json: JsValue): JsResult[ApiVersionSource] = json match {
+      case JsString(RAML.asText)    => JsSuccess(RAML)
+      case JsString(OAS.asText)     => JsSuccess(OAS)
+      case JsString(UNKNOWN.asText) => JsSuccess(UNKNOWN)
+      case e                        => JsError(s"Cannot parse source value from '$e'")
+    }
+
+    def writes(foo: ApiVersionSource): JsValue = {
```

Comment:
Why is this called `foo` which sounds like it could be anything, not a specific type?

Reply:
No idea - it's only in green because of a reformat.  I could change it.....

Reply:
But as it is an arg for a 1 line function and could therefore be x or avs or similar I'm going to leave it as foo.

Reply:
OK, no worries